src/providers/API/storage.ts

```diff
@@ -0,0 +1,12 @@
+import memoryStorage from 'localstorage-memory';
+
+let storage: Storage;
```

Comment:
should we initiate `storage` with the `localStorage` or with `undefined`?

Reply:
storage cannot be undefined, the code was just moved. I assume what does try to use local storage but if is not there switch to the memory, but cannot be undefined.

Reply:
in this case, should we initiate it with `memoryStorage`? I understand what you mean, but as `let storage: Storage;` is just declared, it's already `undefined` in the beginning